entity-framework/core/what-is-new/ef-core-10.0/breaking-changes.md

```diff
@@ -83,3 +86,95 @@ await context.Blogs.ExecuteUpdateAsync(s =>
     }
 });
 ```
+
+<a name="MDS-breaking-changes"></a>
+
+## Microsoft.Data.Sqlite breaking changes
+
+### Summary
+
+| **Breaking change**                                                                                       | **Impact** |
+|:----------------------------------------------------------------------------------------------------------|------------|
+| [Reading DateTimeOffset without an offset now assumes UTC](#DateTimeOffset-read)                          | High       |
+| [Writing DateTimeOffset into REAL column now writes in UTC](#DateTimeOffset-write)                        | High       |
+| [Reading DateTime with an offset now returns value in UTC](#DateTime-read)                                | High       |
+
+### High-impact changes
+
+<a name="DateTimeOffset-read"></a>
+
+#### Reading DateTimeOffset without an offset now assumes UTC
+
+[Tracking Issue #36195](https://github.com/dotnet/efcore/issues/36195)
+
+##### Old behavior
+
+Previously, when reading a `DateTimeOffset` value that did not have an offset (e.g., `2014-04-15 10:47:16`), Microsoft.Data.Sqlite would assume the value was in the local time zone.
+
+##### New behavior
+
+Starting with Microsoft.Data.Sqlite 10.0, when reading a `DateTimeOffset` value that does not have an offset, Microsoft.Data.Sqlite will assume the value is in UTC.
+
+##### Why
+
+Is is to align with SQLite's behavior where timestamps without an offset are treated as UTC.
```

Comment:
Also to remove any dependency on the local configured timezone on the machine where the .NET app happens to be running, no?